core/arch/arm/tee/entry_std.c

```diff
@@ -174,17 +183,16 @@ static TEE_Result copy_in_params(const struct optee_msg_param *params,
 	return TEE_SUCCESS;
 }
 
-static void cleanup_params(const struct optee_msg_param *params,
-			   const uint64_t *saved_attr,
-			   uint32_t num_params)
+static void cleanup_shm_refs(const uint64_t *saved_attr,
```

Comment:
Hi, @jenswi-linaro, 

this change is not covered in the commit message. And I think, it should be moved into separate commit.

Reply:
This is a central part in the commit, especially the previously saved shm_ref should be used instead of reading it again.
Sure, I could delay renaming the function but that is bordering on splitting hairs.

Reply:
I'm talking not about this particular change, but about whole entry_std.c diff.

At least you can mention this explicitly in the commit message. Because right now this patch does two things:

 * it is adding reference counter to `mobj_reg_shm`
 * and it alters `entry_std.c` to make proper use of this reference counter

So, it will be great to see both changes covered in the commit message.

Reply:
Aha, now I get it. I'll split it.

Reply:
Hmm, it has to go in the same commit. I'll expand the commit message as suggested instead.